commands/removerole.js

```diff
@@ -0,0 +1,142 @@
+const Discord = require("discord.js")
+const hutil = require("../utilities/hutil.js")
+const reaction_message_model = require("../models/reaction_roles_message.js")
+const mongoose = require('mongoose');
+mongoose.connect('mongodb+srv://XtratoS:UfTMXyIAxTN6Xr3q@nbot-dg4ad.mongodb.net/rbot?retryWrites=true', { useNewUrlParser: true })
+
+/**
+ * Has 2 usages:
+ * -If a role/emoji is provided, removed that role/emoji from the specified message,
+ * -If no role/emoji provdided, removes all the roles/emojis from the specified message.
+ */
+module.exports = async (bot, message, args) => {
```

Comment:
Unless necessary (i.e. to avoid rebinding of this) arrow functions are discouraged for anything more than a quick one-line filter or transform. Avoid arrow functions for anything more than a one-liner.
```suggestion
module.exports = async function(bot, message, args) {
```